packages/cerebral-debugger/package.json

```diff
@@ -1,38 +1,60 @@
 {
-  "name": "cerebral-debugger",
-  "version": "2.0.0",
+  "name": "electron-quick-start",
+  "version": "1.0.0",
   "private": true,
-  "description": "Cerebral Debugger",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/cerebral/cerebral.git"
+  "description": "A minimal Electron application",
+  "main": "electron/main.js",
+  "scripts": {
+    "start": "DEBUGGER_TARGET=electron NODE_ENV=development concurrently -k 'node ./electron/devServer.js' 'npm run electron'",
+    "electron": "electron .",
+    "build:chromeExtension": "NODE_ENV=production webpack --config webpack.chromeExtension.config",
+    "build:electron": "NODE_ENV=production webpack --config webpack.electron.config && npm run prepare:electron && npm run package:electron",
+    "prepare:electron": "rimraf electron/dist electron/node_modules && copyfiles 'node_modules/ws/**/*' 'node_modules/ultron/**/*' 'node_modules/options/**/*' electron",
+    "package:electron": "electron-packager ./electron CerebralDebugger --platform=darwin --out ./electron/dist"
```

Comment:
Jup, have not looked into that cause I can not test it. So someone else has to look into it :)